modules/viewing-pipeline-run-step-logs.adoc

```diff
@@ -0,0 +1,49 @@
+:_module-type: PROCEDURE
+
+[id="viewing-pipeline-run-step-logs_{context}"]
+= Viewing pipeline run step logs
```

Comment:
Same as previous RE: consistency in 'run step logs' v 'step logs'.

Reply:
Fixed.